h/static/scripts/annotator/plugin/test/pdf-metadata-test.js

```diff
@@ -0,0 +1,73 @@
+'use strict';
+
+var PDFMetadataService = require('../pdf-metadata');
+
+describe('pdf-metadata', function () {
+  var pdfMetadata;
+  var fakePDFViewerApplication = {
+    'documentFingerprint': 'fakeFingerprint',
+    'documentInfo': {
+      'Title': 'fakeTitle',
+    },
+    'metadata': {
+      'metadata': {
+        'dc:title': 'fakeTitle',
+      }
+    },
+    'url': 'fakeUrl',
+  };
+  var sandbox;
```

Comment:
Sandboxes are generally only needed if you are modifying the global environment in some way, eg. by using fake clocks or modifying global properties and later need an easy way to undo those changes after a test runs. If you just need to create a stub in order to test that a particular method was called you can use `sinon.stub()` rather than `sandbox.stub()`.